### ISSUE TITLE
Security definition could not be resolved in Swagger Editor

### ISSUE THREAD

**Issue by dshil** [go-swagger/go-swagger#721]
## Problem statement
I read a doc about generating json spec from code (from comments). 
I follow the example providing in gitbook.
## Swagger specification
```go
// ServeAPI serves the API for this record store
func ServeAPI(host, basePath string, schemes []string) error {

    // swagger:route GET /pets listPets pets users
    //
    // Lists pets filtered by some parameters.
    //
    // This will show all available pets by default.
    // You can get the pets that are out of stock
    //
    //     Consumes:
    //     - application/json
    //     - application/x-protobuf
    //
    //     Produces:
    //     - application/json
    //     - application/x-protobuf
    //
    //     Schemes: http, https, ws, wss
    //
    //     Security:
    //       api_key:
    //       oauth: read, write
    //
    //     Responses:
    //       default: genericError
    //       200: someResponse
    //       422: validationError
    mountItem("GET", basePath+"/pets", nil)
}
```
## Steps to reproduce
Generate doc from code.

```bash
 swagger generate spec -o ./apidoc.json 
```

Put generated json to swagger editor and got:
`Security definition could not be resolved: api_key`
`Security definition could not be resolved: oauth`

**Comment by ychen-symphony** [comment]
Looks like this feature is missing. `api_key` is used in the example but goswagger will not generate the `securityDefinition`

**Comment by shevchenkodenis** [comment]
Confirm this. And there is no way to write securityDefinitions in swagger:meta section.

**Comment by casualjim** [comment]
This was fixed in d659fb1246322b1f42456dec2810ca60d90d84a1
